cmd/log-splicer/main.go

```diff
@@ -20,9 +26,12 @@ import (
 	"go.cryptoscope.co/margaret/legacyflumeoffset"
 )
 
+// TODO: collapse feedinfo + feedjson into just feedinfo? and use temp struct for unmarshaling ID?
```

Comment:
i decided not to do this: feels cleaner the way it is, and the two data structures have their own priorities: FeedInfo collects FeedInfo, FeedJSON is used to persist feed mappings to json. the last one could maybe have a better name, but hey life is short

Reply:
remove the comment then :)